packages/seacas/libraries/ioss/src/cgns/Iocgns_DatabaseIO.C

```diff
@@ -1983,7 +1983,7 @@ namespace Iocgns {
       const Ioss::GroupingEntity *sb   = nb->contained_in();
       int                         zone = Iocgns::Utils::get_db_zone(sb);
       int                         solution_index =
-          Utils::find_solution_index(get_file_pointer(), base, zone, step, CG_Vertex);
```

Comment:
This line should be deleted.  It looks like I accidentally added a duplicate line in my previous PR which was trying to add in your PR changes...